### PR TITLE
fix: add TenTraicion

### LINKED ISSUE #1310: TenTraicion
## Profile Screenshot

screenshot

### PR DESCRIPTION
**Pull request by TenTraicion**
resolves #1310 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `TenTraicion.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by schmelto — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: profiles/TenTraicion.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/103281314/185060662-8b03bbf5-d3dc-4a30-b731-9f5a0424953a.png",
```
Comment (Panquesito7) [inline profiles/TenTraicion.json:2]: I recommend more of an image like this. What do you think?

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/185238850-96892427-0bfe-4cdc-ab7f-a87a80720c47.png",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Comment (TenTraicion) [reply profiles/TenTraicion.json:2]: @Panquesito7, I thought a proper image should be better as it shows the complete design of my GitHub profile. Should I change it to a simpler one like you've provided?

Comment (Panquesito7) [reply profiles/TenTraicion.json:2]: > @Panquesito7, I thought a proper image should be better as it shows the complete design of my GitHub profile. Should I change it to a simpler one like you've provided?

I prefer smaller ones, so it is easier to read and understand, but you can use any size you choose. :)


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Inefficient

**Comment by TenTraicion** [comment]
@Panquesito7 changed as your preference.

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**